slack-base/src/main/java/com/hubspot/slack/client/models/response/migration/MigrationExchangeResponseIF.java

```diff
@@ -0,0 +1,25 @@
+package com.hubspot.slack.client.models.response.migration;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.response.SlackResponse;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface MigrationExchangeResponseIF extends SlackResponse {
+  String getTeamId();
+
+  String getEnterpriseId();
+
+  Map<String, String> getUserIdMap();
+
+  Optional<List<String>> getInvalidUserIds();
```

Comment:
Does Slack always return this array of user IDs? I'm curious if we need `Optional` here? How does the response look like if all the IDs are valid? Is it `"invalid_user_ids": []` or there is no `invalid_user_ids` property at all?

Reply:
When there is invalid user ids slack returns this set of ids. But when all ids are valid slack doesn't return this set at all that's why it is Optional.